docs/api/evm/index.rst

```diff
@@ -0,0 +1,14 @@
+Py-EVM
+======
+
+This section aims to provide a detailed description of all APIs. If you are looking for something more hands-on or higher-level check out the existing `EVM guides </guides/evm>`_.
+
+.. toctree::
+   :maxdepth: 2
+   :name: toc-api-evm
+   :caption: API
+
+   api.chain
+   api.computation
```

Comment:
I think these are currently ordered alphabetically.  **if** hierarchical ordering makes sense it would be `chain -> vm -> state -> computation` but I'm not sure what the best practice here is.

Reply:
IMO, guides should have hierarchical ordering, and API docs should be alphabetical.

Reply:
>IMO, guides should have hierarchical ordering, and API docs should be alphabetical.

I think that makes perfect sense :+1: